src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingMapper.kt

```diff
@@ -168,6 +169,26 @@ object BehandlingMapper {
         )
     }
 
+    fun tilVedtakForFagsystem(behandlinger: List<Behandling>): List<no.nav.familie.kontrakter.felles.klage.FagsystemVedtak> {
+        return behandlinger
+            .filter { it.erAvsluttet }
+            .mapNotNull {
+                val avsluttetDato = it.avsluttetDato ?: error("Mangler avsluttet dato på behandling=${it.id}")
+                val sisteResultat = it.sisteResultat ?: error("Mangler resultat på behandling=${it.id}")
+                if (sisteResultat.erBehandlingHenlagt()) {
+                    return@mapNotNull null
+                }
```

Comment:
bør vi heller ha `.filter(!it.sisteResultat.erBehandlingenHenlagt())` mellom `filter` og `map` ?